travis/getaddons.py

```diff
@@ -71,7 +71,7 @@ def get_addons(path):
         res = [path]
     else:
         res = [os.path.join(path, x)
-               for x in os.listdir(path)
+               for x in sorted(os.listdir(path))
```

Comment:
Could you add a print-screen with returned values using the following `oca_dependencies.txt`:
`enterprise git@github.com:odoo/enterprise.git`